tests/page/page-keyboard.spec.ts

```diff
@@ -464,6 +464,17 @@ it('should dispatch a click event on a button when Enter gets pressed', async ({
   expect((await actual.jsonValue()).clicked).toBe(true);
 });
 
+it('should support simple copy-pasting', async ({ page, isMac }) => {
+  it.skip(!isMac);
```

Comment:
```suggestion
    const modifier = isMac ? 'Meta' : 'Control';
```

And let's use this modifier below to check on other platforms.